shell/src/test/java/org/apache/kafka/shell/glob/GlobVisitorTest.java

```diff
@@ -99,11 +100,11 @@ public MetadataNode child(String name) {
     }
 
     static class InfoConsumer implements Consumer<Optional<MetadataNodeInfo>> {
-        private Optional<List<MetadataNodeInfo>> infos = null;
+        private Optional<List<MetadataNodeInfo>> infos = Optional.empty();
```

Comment:
not sure why we need `Optional` ... it seems empty collection can be equal to `Optional.empty`

Reply:
I made that change just for consistency with the variable declaration. We have it defined as `Optional` and I felt `Optional.empty()` made more sense than `null` here. 
Even the IDE complains here: `Null is used for 'Optional' type in declaration`.
lmk if it makes sense here.
Thanks!